common/constant.go

```diff
@@ -1,8 +1,11 @@
 package common
 
 const (
-	// DefaultGasPriceMultiplier is the default gas price multiplier for outbond txs
-	DefaultGasPriceMultiplier = 2
+	// EVMOuttxGasPriceMultiplier is the default gas price multiplier for EVM-chain outbond txs
+	EVMOuttxGasPriceMultiplier = 1.2
+
+	// BTCOuttxGasPriceMultiplier is the default gas price multiplier for BTC outbond txs
+	BTCOuttxGasPriceMultiplier = 2.0
```

Comment:
I think we should move this in `zetaclient` package. `common` should define types used through the project but here this is specifically a value for ZetaClient

Reply:
Actually, I think this might be a good place for the multiplier; this same multiplier should also be used for creating the TSS migration. outbound  . Whitelist erc20
https://github.com/zeta-chain/zeta-node/blob/7e25cd015c24665e685f329fda371bac01b9ea94/x/crosschain/keeper/msg_server_migrate_tss_funds.go#L110-L110

https://github.com/zeta-chain/zeta-node/blob/7e25cd015c24665e685f329fda371bac01b9ea94/x/crosschain/keeper/msg_server_whitelist_erc20.go#L144-L144

Reply:
>this same multiplier should also be used to create the TSS migration. outbound and Whitelist erc20

What do you mean by this?
The multiplier is used to vote on the gas price, it is therefore already taken into consideration there.
So far the direct usage of the multiplier is only in ZetaClient so I suggest to move it here

Reply:
https://github.com/zeta-chain/node/pull/1881/commits/b396ccb2db2dfdf16cfa02092d66de12b2c15c20

Reply:
okay just saw your comment @ws4charlie 
https://github.com/zeta-chain/node/pull/1881#issuecomment-1999958168

I think that's exactly what I was referring too.Resolving this for now , as this seems out of scope for this pr